safe/gui/tools/help/options_help.py

```diff
@@ -208,7 +208,7 @@ def content():
     bullets.add(m.Text(
         m.ImportantText(tr(
             'Contact email')),
-        tr(' - Use this option to specify the contact person\s email '
+        tr(' - Use this option to specify the contact person\'s email '
```

Comment:
I think we should go with single quote, to keep our coding standard. https://github.com/inasafe/inasafe/wiki/Coding-Standards#strings-and-internationalisation

This string is simple enough (one escape only).